css/style.css

```diff
@@ -0,0 +1,112 @@
+
+body{
```

Comment:
```suggestion
body {
```
Remember about space after the selector.